### DISCUSSION TITLE
Run danger multiple times in same pipeline

### DISCUSSION THREAD

**Discussion by glensc** [danger/danger-js#1256] [Ideas]
Hi,

I'm intertested using danger ci for two purposes (at the same time):
- run some quality tests, display errors, warnings (the usual stuff)
- (ab)use danger to display informative messages

so the qa check runs early in the pipeline possibly failing the whole pipeline, but the information job needs to be run last and use artifacts from previous jobs to render information.

for example reviewapp deploy would leave deploy.json that would contain information about helm deploy name and URLs exposed.

so, how to run danger two instances, so they don't overwrite each other? or maybe some other ideas how to display info bits in merge request? i find danger suitable as it already has integration (tokens, etc) in our infrastructure and has API that I need (just `markdown` method).

**Comment by glensc** [comment]
I found the answer basically:

```
Usage: danger-ci [options]

Runs a Dangerfile in JavaScript or TypeScript.

Options:
  -d, --dangerfile [filePath | remote url]  Specify a custom dangerfile path, remote urls only work with github
  -i, --id [danger_id]                      Specify a unique Danger ID for the Danger run
```

but is there anything else I should be aware of or consider?

**Reply by fbartho** [reply]
This works great. We have two different Dangerfiles in my current team’s repo.
